libclc/amdgcn-amdhsa/libspirv/async/wait_group_events.cl

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <spirv/spirv.h>
+
+_CLC_DEF void _Z23__spirv_GroupWaitEventsjiP9ocl_event(unsigned int scope,
```

Comment:
```suggestion
_CLC_DEF _CLC_OVERLOAD void __spirv_GroupWaitEvents(unsigned int scope,
```

it is also missing the overload for private

Reply:
https://github.com/intel/llvm/blob/sycl/clang/lib/AST/ItaniumMangle.cpp#L2560 is causing different mangling.

In SYCL mode, Default will be 0, so no `UAS0` will be added, by in CL mode, Default is `5` (private) causing the addition of `UAS0`.

This could either be recovered by the remangler (a bit dirty but more scalable) or handled by manual mangling (not great).